TinyConsole/TinyConsoleThreeTapDelegate.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TinyConsoleThreeTapDelegate.swift
+//  TinyConsole
+//
+//  Created by Chang, Hao on 23/12/2016.
+//
+//
+
+import UIKit
+
+/**
+ Implement this protocol to delegate 3 finger tag actions
+ */
+@objc public protocol TinyConsoleThreeTapDelegate: class {
+    /**
+     If the default actions is included (send email & clear console)
+     */
+    @objc optional var isWithDefaultActions: Bool {get}
```

Comment:
You don't need `@objc` and `optional` here. If you would like to implement optional protocols, just implement an default extension for `TinyConsoleThreeTapDelegate` where you implement `isWithDefaultActions`.